wpilibc/src/main/native/cpp/DataLogManager.cpp

```diff
@@ -281,6 +282,10 @@ Instance::Instance(std::string_view dir, std::string_view filename,
   owner.Start(logDir, filename, period);
 }
 
+Instance::~Instance() {
+  owner.GetThreadSharedPtr()->StopNTLog();
```

Comment:
Instead of adding a destructor on the instance, add it to the Thread class.

Reply:
Done.